src/components/mdx/Code.tsx

```diff
@@ -17,25 +17,32 @@ require('prismjs/components/prism-clojure')
 
 // Used to determine if we have to highlight the given index
 // Borrowed from https://prince.dev/blog/highlight-with-react
-const calculateLinesToHighlight = meta => {
+const calculateLinesToHighlight = (meta: string) => {
   const RE = /{([\d,-]+)}/
   if (RE.test(meta)) {
-    const strlineNumbers = RE.exec(meta)[1]
+    const strlineNumbers = RE.exec(meta)?.[1] ?? ''
     const lineNumbers = rangeParser(strlineNumbers)
-    return index => lineNumbers.includes(index + 1)
+    return (index: number) => lineNumbers.includes(index + 1)
   } else {
     return () => false
   }
 }
 
+export interface CodePropsType {
+  className?: string;
+  metastring: string;
+  ['react-live']?: boolean;
+  children: string;
+}
+
 // todo test metastring
-const Code = ({
+const Code: FC<CodePropsType> = ({
   children,
   metastring,
   className = 'language-js',
   ...props
 }) => {
-  const language = className.replace(/language-/, '')
+  const language = className.replace(/language-/, '') as Language;
```

Comment:
That's mostly the hack

Ideally, I would have a function where we mapped the string with `Language` type

Do you think it makes sense or leave as is for now?

Reply:
I think this is totally fine! I'm not too worried :)